### PR TITLE
DRV-262: Fix Repeat() signature

### PR DESCRIPTION
**Pull request by trevorsibanda**
- Make Number parameter optional

### CODE REVIEW

**Comment by faunaee** [comment]
This change makes my example work. 👍

File: faunadb/functions_strings.go [hunk 1]
```diff
@@ -309,12 +309,21 @@ func (fn lTrimFn) String() string { return printFn(fn) }
 //  string - A string concatendanted the specified number of times
 //
 // See: https://app.fauna.com/documentation/reference/queryapi#string-functions
-func Repeat(str, number interface{}) Expr { return repeatFn{Repeat: wrap(str), Number: wrap(number)} }
+func Repeat(str interface{}, number ...interface{}) Expr {
```
Comment (erickpintor) [inline faunadb/functions_strings.go:312]: I think we should adhere to the standard optional interface. Something like:

```go
f.Repeat("foo", f.Number(3))
```
```

**Review by erickpintor — CHANGES_REQUESTED** [review]

**Comment by faunaee** [comment]
The update works too, although there is still a test that fails every time:

```
   --- FAIL: TestRunClientTests/TestEvalContainsFunctions (0.32s)
        client_test.go:1706: 
            	Error Trace:	client_test.go:1706
            	            				client_test.go:1715
            	            				client_test.go:1805
            	Error:      	Not equal: 
            	            	expected: true
            	            	actual  : false
            	Test:       	TestRunClientTests/TestEvalContainsFunctions
```

That test is likely unrelated to this PR's work, but it is something that should be fixed so that the tests pass and everyone can know which builds are good.

**Review by erickpintor — APPROVED** [review]
LGTM!

**Pull request merged**